.github/wiki/Getting-Started.md

```diff
@@ -44,6 +44,25 @@ cd() {
 check_directory_for_new_repository
 ```
 
+By @TheSast
+
+A fish adaptation of the previous script to put in your `$XDG_CONFIG_HOME/fish/functions/cd.fish`:
```

Comment:
Couldn't you use fish's built-in `funcsave` to save the functions to the correct path for each OS?

Reply:
I didn't think of that, I could and I should.
[ I did as you suggested](https://github.com/TheSast/onefetch/commit/b27e54080faa3a5e60f7ef7621d737000c77750a)